pypureclient/flasharray/FA_2_13/client.py

```diff
@@ -18,15 +18,14 @@
 
 
 class Client(object):
-    DEFAULT_TIMEOUT = 15.0
```

Comment:
nit: It's worth mentioning in release notes as a bug fix

Reply:
Will do, good point.